config/manager/default-config/db-statefulset.yaml

```diff
@@ -26,9 +26,9 @@ spec:
               value: /var/lib/pgsql/data
             - name: PGDATA
               value: /var/lib/pgsql/data/userdata
-          envFrom:
-            - secretRef:
-                name: postgres-secrets
+#          envFrom:
+#            - secretRef:
+#                name: postgres-secrets
           image: quay.io/fedora/postgresql-15:latest
```

Comment:
could this container image reference be defined elsewhere, like via an env var in the operator bundle's CSV?

Reply:
It could but for the time the reason is not that clear for me.

Those (like db-statefulset.yaml) yaml files are part of configuration and this configuration is provided with bundle (see bundle/backstage-default-config_v1_configmap.yaml) along with CSV file which is located at the same bundle folder (see bundle/backstage-operator.clusterserviceversion.yaml).  
So, user (Admin) is able to change the image right here and it is not dependent on the operator version, it is a matter of configuration.
So, I am curios, why would we need to make some additional logic for envvar substitution and ask admin to maintain additional envvars instead of just modify the conf file when needed (again, it is not dependent on operator version)?